### PR TITLE
fix ravioli parser

### PR DESCRIPTION
**Pull request by emilhiitola**

### CODE REVIEW

**Comment by maxjvh** [comment]
hey @emilhiitola, thanks for the pull request! seems like there are still some debug logs etc in the code so let me know when you're ready for a review :)

**Comment by emilhiitola** [comment]
Hi @maxjvh, I removed the debug logs. I left in the relevant urls as comments so you can test/add to the database without having to dig around to find them.

File: src/menu-parser/parsers/ravioli.ts [hunk 1]
```diff
@@ -12,64 +13,38 @@ const propertyMap = {
...
+      }*/
+      
+      return {
+        day: date.format('YYYY-MM-DD'),
+        courses: item.description[0]
+          .split('<br>')
+          .filter(c => c.trim().length)
+          .map(title => ({ 
+            title: title, 
+            properties: []}))
```
Comment (maxjvh) [inline src/menu-parser/parsers/ravioli.ts:45]: the properties list still seems to be always empty although properties seem to exist in the feed. to seek inspiration for parsing properties, you could take a look at some of the other parsers.

**Review by maxjvh — CHANGES_REQUESTED** [review]
thanks! just one comment before i could take this in.

File: src/menu-parser/parsers/ravioli.ts (hunk 1 above)
Comment (emilhiitola) [reply src/menu-parser/parsers/ravioli.ts]: Hi, thanks for the request, I modified the parser to properly parse the properties as well

**Comment by maxjvh** [comment]
in production now

**Pull request merged**